src/index.html

```diff
@@ -8,6 +8,13 @@
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h1>Antisnake</h1>
+  <div class="anti-snake container">
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
```

Comment:
give divs a BEM element class 
```suggestion
    <div class="anti-snake__..."></div>
    <div></div>
    <div></div>
    <div></div>
    <div></div>
    <div></div>
```